step-templates/trigger-multitenant-deployment.json

```diff
@@ -0,0 +1,75 @@
+{
+  "Id": "053c7adc-f31f-42e1-b7c7-671d3df55ab2",
+  "Name": "Trigger Multi-Tenant Deployment",
+  "Description": "A step template that triggers Multi-Tenant Deployment. It can be used in not-tenanted projects. The step will lookup most recent release for the specified project if a specific version is not provided. Then it will lookup all tenants that are tagged with the specified tag and create a new deployment for each of the tenants. This will result in multiple tasks scheduled in Octopus. The step is not tracking status of deployment tasks execution.\n\nHypothetical use-case: \n * A multi-tenanted system deployed (as a monoliths) with a non-tenanted Octopus Deploy process.\n * Use this step to start a smoke test for several customers from such a not-tenanted deployment. In this case tenants can be used to represent different customers. This way each customer-specific parameters (access URLs, credentials, etc.) can be configured on tenants level.",
+  "ActionType": "Octopus.Script",
+  "Version": 01,

```

Comment:
Version needs to be a string

Reply:
Thank you for the response. Corrected the version number. All checks seem to have passed.